sql_api/api_instance.py

```diff
@@ -193,6 +194,12 @@ def post(self, request):
             tb_name = query_engine.escape_string(tb_name)
             if resource_type == "database":
                 resource = query_engine.get_all_databases()
+                resource.rows = filter_db_list(
+                    resource.rows, query_engine.instance.show_db_name_regex, True
```

Comment:
```suggestion
                    resource.rows, query_engine.instance.show_db_name_regex, is_match_regex=True
```

建议这么写, 可读性会更好一点, 下面的用法也是类似